### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by sajjadrajaee**
In this PR, I added a home page, quote page, and navbar. use can switch the page using links that work with react-router-dom.

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        {' '}
+        |
+        {' '}
+        <Link className="navItem" to="/">Calculator</Link>
+        {' '}
+        |
+        {' '}
+        <Link className="navItem" to="/quote">quotes</Link>
+      </nav>
+    </div>
```
Comment (Munsa1) [inline src/components/Navbar.js]: - [ ] Upon first loading of the page, the calculator component shows in the middle of the page, what should display is the welcome page as in the screenshot below

Given design
image

Your design
image

- [ ] Also add the demrcating lines to the `navbar` as is in the design
image

File: src/components/Calculator.js [hunk 2]
```diff
@@ -1,41 +1,36 @@
...
-    };
-  }
 
-  EventHandler = (element) => {
+const Calc = () => {
+  const [state, setState] = useState({
+    total: null,
+    operation: null,
+    next: null,
+  });
```
Comment (Munsa1) [inline src/components/Calculator.js:11]: - [ ] On the calculator page the calculator should be aligned to the left of the page as in the given wireframe, kindly fix ir.

Given design
image


Your design
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @sajjadrajaee, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @sajjadrajaee,

Nice job making the required changes. Unfortunately, you missed some of them but you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please fix the positioning of the calculator output to make it on the top of the calculator keys, not on the left side as you did to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| image | image  |

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by BertrandConxy — APPROVED** [review]
Hi @sajjadrajaee, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪
- project requirements are met
- project works fine
- good PR description
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**